modules/util/source-formatter/src/main/resources/dependencies/replacements.json

```diff
@@ -1,4 +1,12 @@
 [
+	{
+		"classNames": [
+			"CountryService"
+		],
+		"from": "fetchCommerceCountry",
+		"issueKey": "LPS-187663",
+		"to": "fetchCountryByA2"
+	},
```

Comment:
@alberttgomes as we talked offline, please add the Refactor of UpgradeJavaCommerceCountryCheck on this PR as well. Should be 4 commits.

- LPS-199343 Update replacements.json
- LPS-199343 Add new test cases to UpgradeCatchAllCheck
- LPS-199343 Remove the UpgradeJavaCommerceCountryCheck
- LPS-199343 Remove the UpgradeJavaCommerceCountryServiceCheck